config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -119,7 +114,6 @@ presubmits:
     branches:
     # The script this job runs is not in all branches.
     - ^master$
-    - ^operator-0.4$
```

Comment:
should the operator tests be part of main instead of release-0.4? 🤔

Reply:
I left it there for now given that's targeting 0.4 branch

Reply:
right but it doesn't make sense to have that work be backported to 0.4 given that it's not even merged in the release-0.4 branch yet...